tests/networkpolicy_test.go

```diff
@@ -12,35 +15,25 @@ import (
 	cd "kubevirt.io/kubevirt/tests/containerdisk"
 )
 
+func pingEventually(fromVmi *v1.VirtualMachineInstance, toIp string) AsyncAssertion {
```

Comment:
This will potentially increase - by a lot - the execution time of failed builds ...

I don't like it.

Reply:
see please https://github.com/kubevirt/kubevirt/pull/4045#issuecomment-681897629

Reply:
doesnt exists anymore, due to what @alonSadan suggested (and you +1), no need for it anymore.